src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal sealed class AssemblyModel
```

Comment:
Could this be auto-serialized, instead of manually generating the XmlNodes and adding children to them like we currently do?